elf/status.py

```diff
@@ -230,7 +230,9 @@ def build_year_status_table(status: YearStatus) -> Table:
 
     for day in sorted(status.days, key=lambda d: d.day):
         # Render stars as a 2-star gauge: ★ for earned, ☆ for missing
-        stars_str = "★" * day.stars + "☆" * (2 - day.stars)
+        stars_str = (
```

Comment:
Personally I don't really need this last commit - I'm interested in the pydantic model output. But it seems like it could be useful for others to render locked days differently.